### DISCUSSION TITLE
How to build older releases

### DISCUSSION THREAD

**Discussion by laeubi** [mono/mono#21679] [Q&A]
I'm currently try to build an older release (because of https://github.com/mono/mono/issues/21676) i tried to download the zip from here https://github.com/mono/mono/tags extracted it and run the usual

```
./autogen.sh --prefix=$PREFIX
make
make install
```

but it fails with 

```
CMake Error at CMakeLists.txt:48 (add_subdirectory):
  The source directory

    /<some path>/mono-mono-5.20.1.34/external/boringssl

  does not contain a CMakeLists.txt file.
```

so what is the recommended approach to build older releases?

**Comment by lextm** [comment]
You need to check out all submodules.
